irr/include/IMeshManipulator.h

```diff
@@ -108,7 +108,7 @@ class IMeshManipulator : public virtual IReferenceCounted
 		if (!mesh)
 			return true;
 		bool result = true;
-		core::aabbox3df bufferbox;
+		core::aabbox3df bufferbox{-1.0f, -1.0f, -1.0f, 1.0f, 1.0f, 1.0f};
```

Comment:
do you intend to remove the default assignment in a later PR, for situations where they are known to be redundant?

Reply:
I could not convince myself that this assignment is redundant. But I do plan to change all the defaults to empty boxes (a behavior change) along with unit tests for each individual case in a separate PR.